README.md

```diff
@@ -58,6 +58,39 @@ If this is your first time running the helper script, make sure to run the `Init
 
 If the pre-commit hook is not letting you commit something and you want to override it. You can ignore the sqlfluff lint hook by running your `git commit` with the `--no-verify` flag.
 
+## Containerizing dbt-cta
+There is a _Dockerfile_ at the root of this repo that can be used to create a dbt-cta image. This image can be used to run dbt as a container, which makes it possible to run dbt in either Docker or Kubernetes. There is a Makefile that can be used to simplify the process of building and running the dbt-cta image locally for testing!
+
+### Makefile Requirements
+- A Google Cloud Platform(GCP) Project with Cloud Build and [Artifact Registry](https://cloud.google.com/artifact-registry/docs/docker) enabled. The Makefile works by using GCP Cloud Build to build and deploy the image to Google's Artifact Registry. 
+- Setup Docker Authentication to Artifact Registry
+    - [Documentation from GCP](https://cloud.google.com/artifact-registry/docs/docker/authentication)
+    - I've had no issues using the [gcloud cli credential helper](https://cloud.google.com/artifact-registry/docs/docker/authentication#gcloud-helper)
+- If you are running the dbt locally, you must have Docker up and running on your computer.
+
+### Using the Makefile
+Below are the commands you can run with the Makefile. To view a summary of these on your CLI, run `make help`
+
+- `make help` - prints out a help menu with all the available commands
+- `make build PROJECT_ID=myprojectid` - Starts a Cloud Build run to build and deploy an image. It uses the current Git HEAD hash as the tag.
+- `make tag PROJECT_ID=myprojectid` - Adds the _latest_ tag to Artifact tagged with the current Git Hash.
```

Comment:
I think I'm seeing a discrepancy with this documentation and current behaviour. I just pushed a tag, but `latest` wasn't applied. 
![screen-shot 2024-12-09 at 2 18 49 PM](https://github.com/user-attachments/assets/e7db8687-efd8-437f-87cc-81b9d1270c3c)

Reply:
Just `make build`. Oh, I see, I would've needed to run make tag to update `latest`?

Reply:
gotcha, makes sense!

ok, I think this is ready to go!